README.md

```
@@ -1,28 +1,125 @@
+    - **Create, Rename, Move, List, and Delete**
+- **File and Folder Sharing**
+    - **Direct Sharing:** Users can directly add other users with specific permissions
+        - **View Only, Edit, and Comment**
+    - **Shareable Links**
+        - **Role Based Permission**
+        - **Password Protection**
+        - **Expiration Date**
+
+<br>and many more...
+
+### 🗺️ System Architecture
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)